production/helm/values.yaml

```diff
@@ -51,7 +51,6 @@ loki:
     #   memory: 128Mi
 
   securityContext:
-    fsGroup: 10001
```

Comment:
You're right, I mixed pod securityContext and container securityContext. I had it at pod level then moved to container so I could get readonly root fs. But fsGroup is necessary otherwise PV will have wrong permissions. I think instead of removing fsGroup, you should move this to be pod level securityContext and move `    readOnlyRootFilesystem: true` to container level.